### DISCUSSION TITLE
List of compatible Matter "Clients" (Apps to control Matter devices)

### DISCUSSION THREAD

**Discussion by Apollon77** [mfucci/node-matter#101] [General]
I think it is a good Idea to start to list Matter compatible Clients/Apps.

The idea is one "Post" per Client, so that we can also have "Client dependend discussions

**Comment by Apollon77** [comment]
So the first "Client" is the "oficial chiptool Android App which is part of the ConnectedHomeIP project and comes together with the chiptool. It is kind of just a bare minimum app that allows to test devices people made

**Comment by Apollon77** [comment]
iOS 16.2 should have a Matter compatible Home App .... Is available in iOS 16.2 Beta already (?)

**Comment by EHylands** [comment]
iOS 16.1 Home App allows you to add Matter accessories.  I can detect a Matter Device created with this library, but i can't complete the pairing process after entering the pin.

**Reply by Apollon77** [reply]
Interesting ... wasn't sure that it is already in 16.1 ... Could you please provide a log from the library for the pairing process? Then we could compare this to one with Chip tool. WHich library version you used? The newest 0.0.10 or older one

**Reply by EHylands** [reply]
I am using latest version 0.0.10.
Matter device get published on network.
iOS is able to find device, but entering pin doesn't create any log in the matter process. 
Will try to compile chip-tool !

MacBook-Pro ~ % matter                                       
2022-9-0 18:2:43.811 INFO Device node-matter@0.0.10
2022-9-0 18:2:43.830 INFO Device Listening

17066BB8F282C5E0._matterc._udp.local.
F018989E8D0E0000.local:5540
192.168.10.101:5540
CM = 1
D = 3840
DN = Matter test device
DT = 257
PH = 33
PI = 
SAI = 300
SII = 5000
T = 1
VP = 65521+32769

![IMG_1360](https://user-images.githubusercontent.com/90003064/198903812-ecfca64b-6b07-41a8-b37d-a01479742f4a.jpeg)

**Reply by Apollon77** [reply]
Will also check it out tomorrow ... But in fact it can be that stuff is mising in 0.0.10 because we still have not implemented all mandatory clusters and such... lets see. But good to know ... but we should handle this as an "issue". COuld yopu pleas eopen one? Then we have this here really for discussions. Thank you!

**Comment by jdesai61** [comment]
chip-tool from [project-chip](https://github.com/project-chip/connectedhomeip/tree/c438b8945e26a84f68ba3608de202e4b939a9080/examples/chip-tool) likely shares lot of code with Android ChipTool app, but it is a distinct client with different usage pattern.
```
chip-tool pairing onnetwork 222 20202021
chip-tool onoff toggle 222 1
chip-tool pairing unpair 222

chip-tool storage clear-all
```
